#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
float32 x
float32 y
float32 z
float32 vel_x
float32 vel_y
float32 vel_z
float32 pitch
float32 heading